clients/admin-ui/cypress/fixtures/generate_dataset.json

```diff
@@ -0,0 +1,958 @@
+{
```

Comment:
Ha, I wondered if this would happen when I created `generat.json`. I think situations like this will come up fairly frequently: multiple fixtures for different examples of the same API. What if make the API name/path a directory and then each of the examples is its own file?

Long term, static fixtures are mainly helpful when copied from an actual API response. If we start making a lot of examples, we're probably better off defining factory functions, like:

```ts
// File: cypress/factories/generate.ts (I made this up, would need to experiment)
// Random strings: https://fakerjs.dev/guide/#node-js
import { faker } from '@faker-js/faker';

export const makeCollection = (opts : Partial<Collection>) => ({
  name: faker.company.bsNoun(),
  // ... other defaults,
  ...opts
})
export const makeOrganization = (opts : Partial<Organization>) => ({
  name: faker.company.companyName(),
  ...opts
})
export const makeGenerate = () => ({
  generate_results: [
    makeOrganization(),
  ]
})
```

Reply:
fancy, yeah that makes sense! for now, I'll put the two `generate` responses in a folder, and if this keeps happening we can try out the factory?